### PR TITLE
BIGTOP-2882: Bump Hadoop to 2.8.1

### PR DESCRIPTION
**Pull request by acmnu**
That patch also bump Hbase to 1.3.1 and Pig to 0.17.0

Pig changes:

    Bump Pig to 0.17.0.
    Remove useless patches PIG-4676 and PIG-4728
    because  it is already in 0.17 release.
    Add patch PIG-5304 to build pig with Hadoop 2.8

Hbase changes:
    There are a number of issues with licenses check when hbase
    building with Hadoop 2.8.x

    That patch bump hbase to latest release and add couple patches.

Hadoop changes:

    fuse_dfs target dir has been changed in 2.8
    HADOOP-12366 has been ported to 2.8.1
    HADOOP-11628 was merged in 2.8.x so removes it.

    Add package goal to mvn. Without that goal tar is incomplete:

    + for dir in '${HADOOP_DIR}/{lib,}' '${HDFS_DIR}/{lib,}' '${YARN_DIR}/{lib,}' '${MAPREDUCE_DIR}/{lib,}'
    + '[' -e /code/build/hadoop/rpm/BUILDROOT/hadoop-2.8.1-243.el7.centos.x86_64/usr/lib/hadoop-mapreduce//hadoop-common-2.8.1.jar ']'
    + exit 1
    error: Bad exit status from /var/tmp/rpm-tmp.5Etrah (%install)
    Bad exit status from /var/tmp/rpm-tmp.5Etrah (%install)

### CODE REVIEW

File: bigtop.bom [hunk 1]
```diff
@@ -165,7 +165,7 @@ bigtop {
     'hbase' {
       name    = 'hbase'
       relNotes = 'Apache HBase'
-      version { base = '1.1.9'; pkg = base; release = 1 }
+      version { base = '1.3.1'; pkg = base; release = 1 }
```
Comment (c0s) [inline bigtop.bom]: This needs to be removed from here: the change is a part of PR #289

File: bigtop.bom [hunk 2]
```diff
@@ -177,7 +177,7 @@ bigtop {
       name    = 'pig'
       pkg     = 'pig'
       relNotes = 'Apache Pig'
-      version { base = '0.15.0'; pkg = base; release = 1 }
+      version { base = '0.17.0'; pkg = base; release = 1 }
```
Comment (c0s) [inline bigtop.bom]: This needs to be removed from here: the change is a part of PR #290

File: bigtop-packages/src/common/hbase/patch1-0001-Partial-backport-HBASE-16712-to-1.3.1.diff [hunk 3]
```diff
@@ -0,0 +1,60 @@
+From 7e3b1d7f830b6e6abd5c4ee6a775f4063b648b60 Mon Sep 17 00:00:00 2001
```
Comment (c0s) [inline bigtop-packages/src/common/hbase/patch1-0001-Partial-backport-HBASE-16712-to-1.3.1.diff]: I am confused: this file is delivered as a part of #289 . Shall it be removed from here?

File: bigtop-packages/src/common/hbase/patch2-0002-Backport-HBASE-17893-to-1.3.diff [hunk 4]
```diff
@@ -0,0 +1,278 @@
+From 1a58e47067d64866917b8834eabc37e4c2887ebb Mon Sep 17 00:00:00 2001
```
Comment (c0s) [inline bigtop-packages/src/common/hbase/patch2-0002-Backport-HBASE-17893-to-1.3.diff]: Same here: a dup of what is done in #289

File: bigtop-packages/src/common/pig/patch0-PIG-5304-Fix-build-issue-with-Hadoop-2.8.diff [hunk 5]
```diff
@@ -0,0 +1,30 @@
+From 65b931966108f374210b83368b8356a79058226d Mon Sep 17 00:00:00 2001
```
Comment (c0s) [inline bigtop-packages/src/common/pig/patch0-PIG-5304-Fix-build-issue-with-Hadoop-2.8.diff]: Already a part of #290

File: bigtop-packages/src/common/pig/patch1-PIG-4676.diff [hunk 6]
```diff
@@ -1,93 +0,0 @@
-diff -ur pig-0.15.0-src_/ivy/libraries.properties pig-0.15.0-src/ivy/libraries.properties
```
Comment (c0s) [inline bigtop-packages/src/common/pig/patch1-PIG-4676.diff]: No need to do it here: the patch is removed in #290

File: bigtop-packages/src/common/pig/patch2-PIG-4728.diff [hunk 7]
```diff
@@ -1,36 +0,0 @@
-diff --git a/ivy.xml b/ivy.xml
```
Comment (c0s) [inline bigtop-packages/src/common/pig/patch2-PIG-4728.diff]: Being removed in #290

**Comment by acmnu** [comment]
@c0s I will close that pull because that was divided on sub-tasks. So It's just a history. See sub-tasks of BIGTOP-2882

**Pull request closed without merging**